conf/steamos-reboot.sh

```diff
@@ -0,0 +1,11 @@
+Cmnd_Alias DESKTOP = /usr/local/sbin/reboot-to-desktop-mode
+Cmnd_Alias STEAMOS = /usr/local/sbin/reboot-to-steamos-mode
+
+%sudo ALL = (root) NOPASSWD:DESKTOP,STEAMOS
+%admin ALL = (root) NOPASSWD:DESKTOP,STEAMOS
+
+Cmnd_Alias REBOOT = /sbin/reboot
+Cmnd_Alias SED = /bin/sed
```

Comment:
We actually shouldn't have passwordless sudo access to `sed` without any parameters. This would allow the steam user to modify any file on the system as root, which isn't a good idea security-wise. 

A fix for this would be to add command aliases for the following exact commands with parameters:
```bash
/bin/sed -i  's/XSession.*/XSession=/g' /var/lib/AccountsService/users/steam
```

```bash
/bin/sed -i  's/XSession.*/XSession=steamos/g' /var/lib/AccountsService/users/steam
```